ontologies/MIRO_sample_image_of_artwork.json

```diff
@@ -0,0 +1,93 @@
+; Comment: next step I assume would be to flesh out necessary context for this, including something to map these more human-readable terms to the ones in the .ttl ontologies
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "identifier": "ia2377yhy",
+  "type": "Work",
+  "sourceIdentifiers": [
+    {
+      "label": "V0006724",
+      "authority": "MIRO"
+    }
+    {
+      "label": "ICV No 6939",
+      "authority": "Videodisk"
+    }
+    {
+      "label": ".b11579328",
+      "authority": "Sierra"
+    }
+  ]
```

Comment:
Ah, right - after the final square bracket, to indicate this is the end of the sourceIdentifiers item in the list and we're moving on to the next?